inlong-common/src/main/java/org/apache/inlong/common/constant/MQType.java

```diff
@@ -15,7 +15,7 @@
  * limitations under the License.
  */
 
-package org.apache.inlong.manager.common.consts;
+package org.apache.inlong.common.constant;
```

Comment:
Why change its package? Suggest adding more info in the description of Issue or PR.